iteration.qmd

```diff
@@ -845,15 +867,16 @@ con |>
 ```
 
 ```{r, include = FALSE}
+# why not show this?
```

Comment:
I lean towards showing clean up code.

Reply:
Because it doesn't actually quiet the message I want it to; so there's something else going on here that I don't understand.